package.json

```diff
@@ -28,7 +28,7 @@
     "package:release": "grunt package:release",
     "test": "grunt jasmine:ng1 && karma start --single-run && grunt e2e",
     "test:e2e": "grunt e2e",
-    "test:karma": "karma start --single-run",
+    "test:karma": "karma start",
```

Comment:
Needs to be reverted. (If you want to run karma outside of npm scripts and without installing it globally, do `npx karma start`.)

Reply:
I'll add that back in and try that thanks.